package.json

```diff
@@ -1,48 +1,53 @@
 {
-  "description": "A Polymer-based web component for images requiring authentication",
+  "name": "@d2l/image",
```

Comment:
It seems like this is a pattern our repos are adopting. Does it make sense to change it?

Reply:
We use the `@d2l` for private packages in CodeArtifact -- unfortunately we don't own the `@d2l` public NPM organization despite our best efforts. Also if you change the name, any consumers will need to be updated.

Reply:
Oooooh, I see. Makes sense.

I think this a breaking change anyways (?), but good to know the context.

On another note...would this make more sense as a UI repo?

Reply:
If anything it's maybe a hypermedia repo due to the token-like nature of it, or maybe something in labs. We've been trying to reserve `BrightspaceUI` for "official"-tier components.